modules/javafx.controls/src/test/java/test/javafx/scene/control/ComboBoxTest.java

```diff
@@ -285,6 +285,29 @@ public Node getDisplayNode() {
         assertNull(comboBox.getSelectionModel());
     }
 
+    @Test public void testNullSelectionModelDoesNotThrowNPEOnValueChange() {
+        ObservableList<String> items = FXCollections.observableArrayList("ITEM1", "ITEM2");
+
+        ListCell<String> buttonCell = new ListCell<>() {
+            @Override
+            protected void updateItem(String item, boolean empty) {
+                super.updateItem(item, empty);
+                setText(item);
+            }
+        };
+        comboBox.setButtonCell(buttonCell);
+        comboBox.setItems(items);
+        comboBox.setSelectionModel(null);
+
+        try {
+            comboBox.setValue(items.get(1));
+        } catch (Exception e) {
+            fail("ComboBox.setValue() should not throw an exception.");
+        }
+
```

Comment:
same as for ChoiceBoxTest

also: there are two locations of the fix (in combo itself and its skin), so there should be 2 separate tests, each concentrating on one - for testing the combo-only, just don't install a skin :)